modules/machine-health-checks-about.adoc

```diff
@@ -6,26 +6,41 @@
 [id="machine-health-checks-about_{context}"]
 = About MachineHealthChecks
 
-MachineHealthChecks automatically repairs unhealthy Machines in a particular
-MachinePool.
+A MachineHealthCheck is a resource which allows users to define conditions under
+which Machines within a Cluster should be considered unhealthy and should be
+automatically remediated.
 
-To monitor machine health, you create a resource to define the
-configuration for a controller. You set a condition to check for, such as
-staying in the `NotReady` status for 15 minutes or displaying a permanent condition
-in the node-problem-detector, and a label for the set of machines to monitor.
+When defining a MachineHealthCheck, users specify conditions to check for and a
+timeout for each of the conditions.
+If any of these conditions is met on the Machine's Node for the duration of the timeout,
+the Machine will be remediated.
+The action of remediating a Machine will trigger a new Machine to be created,
+to replace the failed one.
+
+For example, you may set a condition such as staying in the `NotReady` state
+for 15 minutes or displaying a permanent condition in the node-problem-detector.
 
 [NOTE]
 ====
-You cannot apply a MachineHealthCheck to a machine with the master role.
+Please note that MachineHealthChecks currently *only* support Machines that are
+owned by a MachineSet.
+Please review the xref:#limitations-when-deploying-machinehealthchecks at the bottom of this page for
```

Comment:
Please move this cross reference to an 'Additional resources' section in the assembly module and restructure it to use the absolute `.adoc` path, anchor and context. I have outlined how to do this in detail in another comment in this review.

Reply:
Please see [this link](https://github.com/openshift/openshift-docs/pull/22068#discussion_r496773397) to my other comment.